wallet/wallets/handler.go

```diff
@@ -238,6 +238,34 @@ func (h *Handler) SignTx(name string, req *walletpb.SubmitTransactionRequest, he
 	return commands.NewTransaction(pubKey, marshaledInputData, protoSignature), nil
 }
 
+func (h *Handler) CheckTx(name string, req *walletpb.SubmitTransactionRequest, height uint64, chainID string) (*commandspb.Transaction, error) {
```

Comment:
This handler is used by the old API v1. We don't port feature to v1.

Reply:
Yes, I have removed this as well now.
Thanks!